File: Level1/K번째수/Solution.js
function solution(array, commands) {
    var answer = [];
    var arr = [];
    for (let i = 0; i < commands.length; i++) {
        arr = array.slice(commands[i][0] - 1, commands[i][1]);
        answer.push(arr.sort((a, b) => {
            return a - b;
          })[commands[i][2] - 1]);
    }
    return answer;
}

File: Level1/K번째수/Solution.java
package K번째수;
import java.util.*;
public class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int i = 0;
        for (int[] command : commands) {
            int[] new_array = Arrays.copyOfRange(array, command[0]-1, command[1]);
            Arrays.sort(new_array);
            answer[i] = new_array[command[2] - 1];
            i++;
        }

        return answer;
    }
}